modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -434,20 +495,22 @@
 			attacking_item.name = "[clothing_reagent.name] [attacking_item.name]"
 		attacking_item.color = mix_color_from_reagents(attacking_item.reagents.reagent_list)
 		to_chat(user, span_notice("You finish imbueing [attacking_item]..."))
+		user.mind.adjust_experience(/datum/skill/smithing, 30) //successfully imbueing will grant great experience!
 		playsound(src, 'sound/magic/demon_consume.ogg', 50, TRUE)
 		in_use = FALSE
-		return
+		return TRUE
 
 	if(istype(I, /obj/item/ceramic))
 		var/obj/item/ceramic/ceramic_item = I
-		if(forge_temperature <= 50)
+		var/actioning_speed = HAS_TRAIT(user, TRAIT_CERAMIC_MASTER) ? MASTER_TIMED : DEFAULT_TIMED
```

Comment:
this var name is confusing and doesnt convey its ceramic-based; try like, ceramics_speed or something